pom.xml

```diff
@@ -46,7 +46,7 @@
         <maven.compiler.source>17</maven.compiler.source>
         <maven.compiler.target>17</maven.compiler.target>
 
-        <testcontainers.version>1.18.2</testcontainers.version>
+        <testcontainers.version>1.18.1</testcontainers.version>
```

Comment:
Virker det ikke med 1.18.2?

Reply:
Kanskje du heller kan prøve 1.18.3?

Reply:
Det funket ikke hos meg lokalt med 1.18.2 men tror det funket fint her på github så hadde forsåvidt ikke trengt å committe endringen hvis alt fungerer for dere andre? ( skal sjekke med 1.18.3 også)

Reply:
Den nye versjonen funka :)